mysql/src/main/kotlin/com/huanshankeji/exposedvertxsqlclient/mysql/exposed/ExposedDatabases.kt

```diff
@@ -13,7 +13,7 @@ import java.sql.Connection
  * @see exposedDatabaseConnect
  */
 @ExperimentalEvscApi
-fun ConnectionConfig.Socket.exposedDatabaseConnectMysql(
+fun ConnectionConfig.Socket.exposedDatabaseConnectMySQL(
```

Comment:
In our own code we use the spelling "MySql", which follows the official Kotlin naming conventions and is more consistent.